### PR TITLE
Added tests for Video

### PR DESCRIPTION
**Pull request by ShimiSun**
#### What does this PR do?
Added tests for Video

#### Where should the reviewer start?
video.js
#### What testing has been done on this PR?
Jest
#### How should this be manually tested?
Jest + review
#### What are the relevant issues?
#4539 
#### Screenshots (if appropriate)
image

#### Do the grommet docs need to be updated?
Nope
#### Should this PR be mentioned in the release notes?
Nope
#### Is this change backwards compatible or is it a breaking change?
b/c

### CODE REVIEW

File: src/js/components/Video/Video.js [hunk 1]
```diff
@@ -182,7 +182,11 @@ const Video = forwardRef(
           const scrubberRect = scrubberRef.current.getBoundingClientRect();
           const percent =
             (event.clientX - scrubberRect.left) / scrubberRect.width;
-          videoRef.current.currentTime = duration * percent;
+          if (!duration)
```
Comment (ericsoderberghp) [inline src/js/components/Video/Video.js:185]: The use of curly braces is a bit inconsistent in this if/else.

File: src/js/components/Video/Video.js [hunk 2]
```diff
@@ -182,7 +182,11 @@ const Video = forwardRef(
...
+            console.warn('The video provided has no valid duration property.');
```
Comment (ericsoderberghp) [inline src/js/components/Video/Video.js:186]: Is there a chance that the video hasn't been loaded from the server yet? If so, I don't know that we'd want to add this message into the browser console.

Comment (ShimiSun) [reply src/js/components/Video/Video.js:186]: Not sure if a server error will land in this space, but in some cases, you might have a malfunctioning video that has an issue with its duration.

File: src/js/components/Video/Video.js (hunk 1 above)
Comment (ShimiSun) [reply src/js/components/Video/Video.js]: done.

File: src/js/components/Video/Video.js (hunk 2 above)
Comment (ericsoderberghp) [reply src/js/components/Video/Video.js:186]: I wasn't thinking there was a server error, just that the latency or bandwidth was slower. I wouldn't want to show a console warning in that case. I think that's more likely than a malfunctioning video.

Comment (ShimiSun) [reply src/js/components/Video/Video.js:186]: So, should I change the message to:
"The video provided has no valid duration property, check your server connection."
Or you are suggesting not to have a warning at all?

Comment (ericsoderberghp) [reply src/js/components/Video/Video.js:186]: no warning

Comment (ShimiSun) [reply src/js/components/Video/Video.js:186]: In this case, the next line would break the code and send an error, are we good with it?

Comment (ericsoderberghp) [reply src/js/components/Video/Video.js:186]: No, I was thinking we'd do:
```
if (duration) videoRef.current.currentTime = duration * percent;
```

Comment (ShimiSun) [reply src/js/components/Video/Video.js]: Thanks. ready.

**Review by ericsoderberghp — APPROVED** [review]

**Pull request merged**